### PR TITLE
Improving blackfire docs

### PR DESCRIPTION
**Pull request by matthiaz**
Going through the documentation on how to setup blackfire I noticed some things that can be improved.

### CODE REVIEW

File: docs/src/integrations/profiling/blackfire.md [hunk 1]
```diff
@@ -62,35 +62,45 @@ Blackfire enables to have a fine grained configuration of server credentials acr
 Configuring server credentials on your master branch will enable you to make sure you can profile any other branch:
 
 ```bash
-platform variable:create -e master env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
-platform variable:create -e master env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
+platform variable:create -e master --json=false --visible-build=false --sensitive=true --level=project env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
```
Comment (lolautruche) [inline docs/src/integrations/profiling/blackfire.md:65]: It is actually not a sensitive variable 😉

**Review by lolautruche — CHANGES_REQUESTED** [review]
Thank you @matthiaz!
By the way, you may check the presence and version of Blackfire probe by running php -v 😉

File: docs/src/integrations/profiling/blackfire.md (hunk 1 above)
Comment (chadwcarlson) [inline docs/src/integrations/profiling/blackfire.md:65]:
```suggestion
platform variable:create -e master --json=false --visible-build=false --level=project env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
```

File: docs/src/integrations/profiling/blackfire.md [hunk 2]
```diff
@@ -62,35 +62,45 @@ Blackfire enables to have a fine grained configuration of server credentials acr
...
 
 #### Configuring server credentials per branch
 
 A recommendation is to have a [Blackfire environment](https://blackfire.io/docs/reference-guide/environments#documentation) for production, another one for staging, and another one for development/integration. That can be mapped in Platform.sh to one Blackfire environment for the production branch, one for the staging branch, and one for all feature branches.
 
+
 ```bash
-platform variable:create -e=<insert your branch name> env:BLACKFIRE_SERVER_ID <insert your Server ID>
-platform variable:create -e=<insert your branch name> env:BLACKFIRE_SERVER_TOKEN <insert your Server Token>
+platform variable:create -e=<insert your branch name> --json=false --visible-build=false --sensitive=true --level=environment env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
```
Comment (chadwcarlson) [inline docs/src/integrations/profiling/blackfire.md:75]:
```suggestion
platform variable:create -e=<insert your branch name> --json=false --visible-build=false --level=environment env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
```

File: docs/src/integrations/profiling/blackfire.md [hunk 3]
```diff
@@ -62,35 +62,45 @@ Blackfire enables to have a fine grained configuration of server credentials acr
 Configuring server credentials on your master branch will enable you to make sure you can profile any other branch:
 
 ```bash
-platform variable:create -e master env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
-platform variable:create -e master env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
+platform variable:create -e master --json=false --visible-build=false --level=project env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
+platform variable:create -e master --json=false --visible-build=false --sensitive=true --level=project env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
```
Comment (lolautruche) [inline docs/src/integrations/profiling/blackfire.md:66]:
```suggestion
platform variable:create -e master --json=false --visible-build=false --sensitive=false --level=project env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
platform variable:create -e master --json=false --visible-build=false --sensitive=false --level=project env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
```

File: docs/src/integrations/profiling/blackfire.md [hunk 4]
```diff
@@ -62,35 +62,45 @@ Blackfire enables to have a fine grained configuration of server credentials acr
...
 #### Configuring server credentials per branch
 
 A recommendation is to have a [Blackfire environment](https://blackfire.io/docs/reference-guide/environments#documentation) for production, another one for staging, and another one for development/integration. That can be mapped in Platform.sh to one Blackfire environment for the production branch, one for the staging branch, and one for all feature branches.
 
+
 ```bash
-platform variable:create -e=<insert your branch name> env:BLACKFIRE_SERVER_ID <insert your Server ID>
-platform variable:create -e=<insert your branch name> env:BLACKFIRE_SERVER_TOKEN <insert your Server Token>
+platform variable:create -e=<insert your branch name> --json=false --visible-build=false --level=environment env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
+platform variable:create -e=<insert your branch name> --json=false --visible-build=false --sensitive=true --level=environment env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
```
Comment (lolautruche) [inline docs/src/integrations/profiling/blackfire.md:76]:
```suggestion
platform variable:create -e=<insert your branch name> --json=false --visible-build=false --level=environment --sensitive=false env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
platform variable:create -e=<insert your branch name> --json=false --visible-build=false --sensitive=false --level=environment env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
```

**Review by lolautruche — CHANGES_REQUESTED** [review]

**Comment by lolautruche** [comment]
The branch needs to be rebased before merging 😉

**Comment by chadwcarlson** [comment]
@lolautruche resubmitted here for less of a mess https://github.com/platformsh/platformsh-docs/pull/1854

**Pull request closed without merging**